.github/workflows/build.yml

```diff
@@ -0,0 +1,74 @@
+name: Python package
+
+on: [pull_request]
+
+env:
+  INSTALL_DIR: ${{ github.workspace }}/install
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-latest]
+        include:
+        - python-version: 3.6
+          os: macos-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      env:
+        LD_LIBRARY_PATH: "${{ env.INSTALL_DIR }}/lib:${{ env.INSTALL_DIR }}/lib64"
+        DYLD_LIBRARY_PATH: "${{ env.INSTALL_DIR }}/lib:${{ env.INSTALL_DIR }}/lib64"
+        PYTHONPATH: ${{ env.INSTALL_DIR }}/lib/python${{ matrix.python-version }}/site-packages:${{ env.INSTALL_DIR }}/lib/python${{ matrix.python-version }}/dist-packages
+      run: |
```

Comment:
Replace bash `if` with:

```yaml
- name: Install Ubuntu Dependencies
  if: ${{ matrix.os == "ubuntu-latest" }}
  run: |
    sudo apt-get install -y ...
```

I'm not entirely certain about the syntax, but let's avoid doing bash `if`s.